content/en/02-installing-pixie/02-setting-up-k8s/01-minikube-setup.md

```diff
@@ -38,7 +38,7 @@ To deploy a specific K8s version [supported by Pixie](/installing-pixie/requirem
 
 ## Verify
 
-Run `minikube dashboard` or `kubectl get nodes` to verify your cluster is up and running.
+Run `kubectl get nodes` to verify your cluster is up and running. Run 'kubectl config current-context` to verify that K8s is pointing to your cluster. 
```

Comment:
I know k8 is an abbreviation for kubernetes but I think it's probably better to just be explicit here and say `the Kubernetes command-line tool` or `kubectl`.